### PR TITLE
Fixes #21928 - add uuid index to katello_subscription_facets

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: db/migrate/20171211124439_add_uuid_index_to_katello_subscription_facets.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUuidIndexToKatelloSubscriptionFacet < ActiveRecord::Migration
```
Comment (jlsherrill) [inline db/migrate/20171211124439_add_uuid_index_to_katello_subscription_facets.rb]: The class name doesn't match the filename  (Facet vs Facets), causing the tests to fail.

Comment (tbrisker) [reply db/migrate/20171211124439_add_uuid_index_to_katello_subscription_facets.rb]: oops, fixed.

**Comment by jlsherrill** [comment]
[test rails-5]

**Comment by jlsherrill** [comment]
[test rails-5]

**Comment by jlsherrill** [comment]
[test default]

**Pull request merged**